packages/theatre/src/lib/theatre/Theatre.svelte

```diff
@@ -11,10 +11,7 @@
   export let config: IProjectConfig | undefined = undefined
 </script>
 
-<Studio
-  enabled={studio.enabled}
-  hide={studio.hide}
->
+<Studio {...studio}>
```

Comment:
This fixes another bug if the component looks like this:
```svelte
<Theatre studio={{ hide }}>
```

Enabled was passed explicitly, and therefore svelte was setting it as `undefined` rather than the default `true`:

```svelte
<script>
export let enabled = true // is set as undefined
...
</script>

<!-- will not mount because enabled is undefined -->
{#if browser && enabled}
  <InnerStudio {hide}>
    <slot />
  </InnerStudio>
{:else}